resources/resources.go

```diff
@@ -673,3 +673,12 @@ func VtoR(v pgraph.Vertex) Res {
 //	}
 //	return nil
 //}
+
+// Lookup returns a pointer to the resource's struct.
```

Comment:
Not sure why we need a helper function for this.

Reply:
Just mirroring funcs Lookup here

Reply:
also registeredResources is private :)

Reply:
I didn't realize this was in resources.go :P